src/workers/postTranslated.ts

```diff
@@ -15,24 +15,26 @@ export const postTranslated: TypedWorker<'kvasir.v1.post-translated'> = {
     }
 
     try {
-      await con
+      const query = con
         .getRepository(Post)
         .createQueryBuilder()
         .update(Post)
         .set({
-          translation: () => `jsonb_set(
+          translation: () => /*sql*/ `jsonb_set(
           translation,
-          :language,
-          :translations::jsonb,
+          :languages,
+          COALESCE(translation->:language, '{}'::jsonb) || :translations::jsonb,
           true
         )`,
         })
         .setParameters({
-          language: [language],
+          language,
+          languages: [language],
```

Comment:
Couldn't figure out how to get `:translaion` to be what I wanted. So easy fix is just creating a new param that is the array of translation.

Reply:
Nvm, fixed it in 514f97b13ee793d6c3a59986ee3f4850be18719c